kafka/resource_kafka_acl.go

```diff
@@ -60,15 +61,22 @@ func kafkaACLResource() *schema.Resource {
 				ForceNew: true,
 			},
 		},
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(10 * time.Minute),
+			Read:   schema.DefaultTimeout(10 * time.Minute),
+			Update: schema.DefaultTimeout(10 * time.Minute),
+			Delete: schema.DefaultTimeout(10 * time.Minute),
+		},
```

Comment:
Standard terraform CRUD timeouts
https://www.terraform.io/plugin/sdkv2/resources/retries-and-customizable-timeouts